### PR TITLE
Dlib 19.8

### PR DESCRIPTION
**Pull request by xsacha**

### CODE REVIEW

File: cmake/projects/dlib/hunter.cmake [hunk 1]
```diff
@@ -13,11 +13,22 @@ hunter_add_version(
     PACKAGE_NAME
     dlib
     VERSION
-    "19.6-p1"
```
Comment (ruslo) [inline cmake/projects/dlib/hunter.cmake]: Please keep old version.

Comment (xsacha) [reply cmake/projects/dlib/hunter.cmake]: Oops, missed that

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3202
* https://travis-ci.org/ingenue/hunter/builds/326236625?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3203
* https://travis-ci.org/ingenue/hunter/builds/326240228?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.222

**Pull request merged**